aedes.js

```diff
@@ -36,12 +36,13 @@ function Aedes (opts) {
 
   opts = Object.assign({}, defaultOptions, opts)
 
-  this.id = opts.id || shortid()
+  this.id = opts.id || uuidv5('https://github.com/mcollina/aedes', uuidv5.URL)
```

Comment:
will this be different at every run? That's the reason why we use short-ids in the beginning.

Reply:
Yes, the id will be different at every run. The parameter in uuidv5 is a namespace.
https://www.npmjs.com/package/uuid